diff --git a/ports/unix/mpbthciport.c b/ports/unix/mpbthciport.c
index 765146677..8797d4146 100644
--- a/ports/unix/mpbthciport.c
+++ b/ports/unix/mpbthciport.c
@@ -142,9 +142,10 @@ static int configure_uart(void) {
     toptions.c_cc[VMIN] = 1;
     toptions.c_cc[VTIME] = 0;
 
-    // Enable HW RTS/CTS flow control.
+    // Disable SW flow control.
     toptions.c_iflag &= ~(IXON | IXOFF | IXANY);
-    toptions.c_cflag |= CRTSCTS;
+    // Disable HW RTS/CTS flow control (not supported by USB CDC VCP).
+    toptions.c_cflag &= ~CRTSCTS;
 
     // 1Mbit (TODO: make this configurable).
     speed_t brate = B1000000;
